wireguard/types.go

```diff
@@ -46,9 +46,11 @@ func NewNCIface(host *config.Config, nodes config.NodeMap) *NCIface {
 	if config.Netclient().ProxyEnabled && len(peers) > 0 {
 		peers = peer.SetPeersEndpointToProxy(peers)
 	}
+	iface := netmaker.Iface // store current iface cfg before it gets overwritten
```

Comment:
Is this going to work the first time this function is called? wont iface be empty?

Reply:
first time it gets created and this variable will be updated via creation